src/common/KokkosKernels_SparseUtils_cusparse.hpp

```diff
@@ -73,11 +73,16 @@ inline void cusparse_internal_safe_call(cusparseStatus_t cusparseStatus,
   }
 }
 
+  // The macro below defines is the public interface for the safe cusparse calls.
+  // The functions themselves are protected by impl namespace.
+#define KOKKOS_CUSPARSE_SAFE_CALL(call) \
```

Comment:
Thank you for namespacing the macro : - )

Reply:
No problem, this should allow you and others to use it if needed in other packages that rely on kk